systems/olcf-frontier/system.py

```diff
@@ -0,0 +1,393 @@
+# Copyright 2023 Lawrence Livermore National Security, LLC and other
+# Benchpark Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import pathlib
+
+from benchpark.directives import variant
+from benchpark.system import System
+from packaging.version import Version
+
+
+class OlcfFrontier(System):
+
+    variant(
+        "compiler",
```

Comment:
@hagertnl "cce18.0.1" should be "cce@18.0.1"

Reply:
Fixed.